client/src/app/pages/applications/applications-table/applications-table.tsx

```diff
@@ -610,13 +609,13 @@ export const ApplicationsTable: React.FC = () => {
 
   const dropdownItems = [...importDropdownItems, ...applicationDropdownItems];
 
-  const isAnalyzingAllowed = () => {
+  const isAnalyzingDisabled = () => {
     const candidateTasks = selectedRows.filter(
       (app) =>
         !tasks.some(
           (task) =>
             task.application?.id === app.id &&
-            task.state?.match(/(Created|Running|Ready|Pending)/)
+            task.state?.match(/(Succeeded|Failed)/)
```

Comment:
Same here, could switch the array includes check instead of the regex.

Reply:
Agreed on switching from regex being the better fit.